elide-model-config/src/test/java/com/yahoo/elide/modelconfig/validator/DynamicConfigValidatorTest.java

```diff
@@ -140,8 +140,7 @@ public void testInheritanceCycle() throws Exception {
             assertEquals(2, exitStatus);
         });
 
-        assertTrue(error.contains("Inheriting from table"));
-        assertTrue(error.contains("creates an illegal cyclic dependency."));
+        assert ("Inheriting from table 'A' creates an illegal cyclic dependency.\n".equals(error) || "Inheriting from table 'B' creates an illegal cyclic dependency.\n".equals(error));
```

Comment:
This is resolved now in elide-5.x.  You can just rebase your PR to pick up the fix.